### PR TITLE
RUCK-11 Create Action Recogniser Model

### PR DESCRIPTION
**Pull request by erf300three**
We have create an action recognition model which is able to take in short 16 frame clips and then determine if a tackle has occurred within those frames. This model has been trained on a custom dataset which includes fully labelled 16 frame clips stating whether a part of a tackle occurs in at least half of the frames. 

### Results 
We have seen better than expected results with this model with an accuracy of around 90% with an F1 score of 0.5. This is more impressive as we are working with a 1:10 class imbalance between the positive and negative class. 

### Training 
We trained the model using the training set that we trained with an early stop of 20 epochs using the data augmentation techniques of random horizontal flip and random rotation to prevent overfitting and make the model perform better on unseen data.

### CODE REVIEW

**Comment by erf300three** [comment]
We have tested this on full length clips but they are unable to be uploaded here and will hopefully be made available at the end of this project. The model is working as hoped though more time needs to be spent refining this model to see if it can be made more accurate and the effects of different hyperparameters such as the number of frames that are passed in.
